Marathi/mazi_marathi.txt

```diff
@@ -0,0 +1 @@
+मराठी माझी मातृभाषा आहे आणि मला त्याचा अभिमान आहे
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.